pgml-dashboard/notebooks/static/notebooks/js/notebook.js

```diff
@@ -71,10 +66,18 @@ export default class extends Controller {
     this.notebookNameTarget.classList.add('hidden')
   }
 
-  detectSql(codeMirror) {
```

Comment:
You need to run this when the cell is edited because it defaults to `sql` even if it's markdown, enabling wrong syntax highlighting.